operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -0,0 +1,78 @@
+## HPE PDU Admin Procedures
+* [Connecting to HPE PDU Web Interface](#connecting-to-hpe-pdu-web-interface)
+* [HPE PDU Initial Set-up](#hpe-pdu-initial-set-up)
+* [Updating HPE PDU Firmware](#updating-hpe-pdu-firmware)
+* [Changing HPE PDU User Passwords](#changing-hpe-pdu-user-passwords)
+* [Discovery of HPE PDU after upgrading CSM](#discovery-of-hpe-pdu-after-upgrading-csm)
+
+**Because of the pulling method used to process sensor data from the HPD PDU, telemetry data may take up to 6 minutes to refresh, this includes outlet status, voltage and other data from the PDU.**

```

Comment:
I would be more explicit here about outlet power status. Say something like "...this includes outlet power status in HSM..." instead of "...this includes outlet status..."

Reply:
Updated.